### ISSUE TITLE
Function shift() does not match InfluxDB 2 spec

### ISSUE THREAD

**Issue by hinst** [influxdata/influxdb-client-java#235]
__Steps to reproduce:__
Build a query using class `Flux`, function `shift`

__Expected behavior:__

Function `timeShift` will be outputted, matching InfluxDB 2 spec: https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/timeshift/

__Actual behavior:__
Function `shift` will be outputted:
```
|> shift(shift:0s)
```
However, it does not match the spec. InfluxDB says: undefined identifier `shift`

__Specifications:__
 - Client Version: 2.3.0
 - InfluxDB Version: 2
 - JDK Version: 11
 - Platform: Client=Windows, Server=Ubuntu

**Comment by bednar** [comment]
Hi @hinst,

thanks for using our client.

I've prepare the PR - #236 to fix this issue. Stay tuned.

Regards

**Comment by bednar** [comment]
Hi @hinst,

the issue is fixed in [2.4.0 milestone](https://github.com/influxdata/influxdb-client-java/milestone/23). You could use a preview version by:

```xml
<dependency>
  <groupId>com.influxdb</groupId>
  <artifactId>influxdb-client-flux</artifactId>
  <version>2.4.0-SNAPSHOT</version>
</dependency>
```

Regards
